org.nongnu.gsequencer.gsequencer.json

```diff
@@ -0,0 +1,233 @@
+{
+    "app-id": "org.nongnu.gsequencer.gsequencer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gsequencer",
+    "rename-desktop-file": "gsequencer.desktop",
```

Comment:
I would use also a `"copy-icon": true` here.

Reply:
These should be renamed upstream.